kubernetes_state/datadog_checks/kubernetes_state/data/auto_conf.yaml

```diff
@@ -18,3 +19,12 @@ instances:
     #    label_to_match: deployment
     #    labels_to_get:
     #      - label_addonmanager_kubernetes_io_mode
+    #
+    # By default the hostname for metrics containing the node label will be
+    # overriden by the value of the label, this can be deactivated (all metrics
+    # will be attached to the host running KSM)
+    # hostname_override: true
+    #
+    # Send (deprecated) pod.phase service checks (default true, will be turned off in future versions)
+    # We recommend using the gauges instead, which will benefit from history
```

Comment:
the **corresponding** gauges, maybe also add their name or at least one of them.

Reply:
which ~will~ benefit from **historical data unlike service checks**